src/test/java/br/com/six2six/fixturefactory/function/NumberSequenceFunctionTest.java

```diff
@@ -41,7 +44,8 @@ public void doubleSequence() {
 		SequenceFunction function = new SequenceFunction(new NumberSequence(1.23d, 2));
 		
 		for (int i=1; i<=5; i=i+2) {
-			assertEquals("doubles should be equal", function.generateValue(), (double) i+(.23d));
+			Double value = Double.valueOf(i+(.23d));
+			assertEquals("doubles should be equal", function.generateValue(), value);
 		}
 	}
```

Comment:
@mundodojava could you explain the reason for these changes at NumberSequenceFunctionTest.java?  
Did you find any issue with the autoboxing/unboxing usage?

Thank you!

Reply:
fixed